tilequeue/process.py

```diff
@@ -645,7 +683,7 @@ def convert_source_data_to_feature_layers(rows, layer_data, unpadded_bounds, zoo
         # expecting downstream in the process_coord function
         for layer_datum in layer_data:
             layer_name = layer_datum['name']
-            layer_props = row_props_by_layer[layer_name]
+            layer_props = row_props_by_layer.get(layer_name)
```

Comment:
Any side-effect this might introduce?

Reply:
i dont think so. if you scroll up just a bit you'll see that this object has props for every layer except this new landmarks layer. those are set to either a value or to `None`. now there isnt one for landmarks so changing this to `get` will just add handling for landmarks not existing above. alternatively i could have added landmarks above and just set it to `None` but there are a lot of places in the code that say something like "TODO: stop hardcoding all the layers" so i didnt want to add to the mess.